### DISCUSSION TITLE
功能建议1

### DISCUSSION THREAD

**Discussion by mieyeaa** [anyproto/contributors#37] [Q&A]
anytype是否支持网页嵌套网页，然后子网页再嵌套网页;网页嵌套网页可以不停的嵌套下去。

**Comment by fuksman** [comment]
Hello @mieyeaa!

Please see our [contributing guidelines](https://github.com/anyproto/contributors/blob/b51fdb2ed23508ade4caec79199863caa293ed22/docs/CONTRIBUTING.md).

We prefer all communications to be in English.

Also, if I got it correctly, your question is not technical and it's about how to use Anytype. [community.anytype.io](community.anytype.io) can be a better place to ask it. If I am wrong and the question is related to contributing, please create a new discussion following [contributing guidelines](https://github.com/anyproto/contributors/blob/b51fdb2ed23508ade4caec79199863caa293ed22/docs/CONTRIBUTING.md).
